scalableminds/webknossos
# Save on merge and show spinner while uploading nml to merge (fix #1092)

For the large code block that appears removed and added again:
- I indented it because I wrapped it in a Spin element.
- I added the `beforeUpload` prop to the NML upload element

### Mailable description of changes:
 - The tracing will definitely be saved before merging and a Spinner will be displayed while uploading an nml to merge.

### Steps to test:
- Open a tracing and open the merge modal. Upload a large nml - a spinner should be displayed while uploading, which should be removed when the upload is finished or failed.
- Merge with unsaved changes and check whether the merged tracing contains those unsaved changes

### Issues:
- fixes #1092

------
